docs/understand/sizing-guide.md

```diff
@@ -62,42 +62,29 @@ in the range of 150 MiB per every 100,000 uncommitted writes.
 
 We recommend starting at 10 GiB for a production deployment, as it will likely be more than enough.
 
-#### RAM
-Since the data size is small, it's recommended to provide enough memory to hold the vast majority of that data in RAM.
-Cloud providers will save you the need to tune this parameter - it will be set to a fixed percentage the chosen instance's available RAM (25% on AWS RDS, 30% on Google Cloud SQL).
-It is recommended that you check with your selected cloud provider for configuration and provisioning information for you database.
-For self-managed database instances follow these best practices
-
 <div class="tabs">
   <ul>
     <li><a href="#postgres-ram">PostgreSQL</a></li>
     <li><a href="#dynamodb-ram">DynamoDB</a></li>
   </ul>
 <div markdown="1" id="postgres-ram">
+#### RAM
+Since the data size is small, it's recommended to provide enough memory to hold the vast majority of that data in RAM.
```

Comment:
Thanks!

I think that both drivers should have the same sections so that the page will look consistent.
We can say at the RAM and CPU sections of dynamodb that since this is a managed service - AWS will handle these

Reply:
done